x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -170,10 +182,37 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
       sortable: false,
       textOnly: true,
     },
+    {
+      'data-test-subj': 'aiopsSpikeAnalysisGroupsTableColumnPValue',
+      width: NARROW_COLUMN_WIDTH,
+      field: 'pValue',
+      name: (
+        <EuiToolTip
+          position="top"
+          content={i18n.translate(
+            'xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.pValueColumnTooltip',
+            {
+              defaultMessage:
+                'The significance of changes in the frequency of values; lower values indicate greater change',
+            }
+          )}
+        >
+          <>
+            <FormattedMessage
+              id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.pValueLabel"
+              defaultMessage="p-value"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </>
+        </EuiToolTip>
+      ),
+      render: (pValue: number | null) => pValue?.toPrecision(3) ?? NOT_AVAILABLE,
```

Comment:
Nit, Could this function be moved to a common utils area to reduce the duplication of the `NOT_AVAILABLE` string?
I see it's defined three times.

Reply:
Good suggestions, we'll pick this up in follow-up, there's quite some duplicate code in the components for the regular table, the grouped table and its expanded row now.